astroplan/plots/finder.py

```diff
@@ -13,7 +13,8 @@
 @u.quantity_input(fov_radius=u.deg)
 def plot_finder_image(target, survey='DSS', fov_radius=10*u.arcmin,
                       log=False, ax=None, grid=False, reticle=False,
-                      style_kwargs=None, reticle_style_kwargs=None):
+                      style_kwargs=None, reticle_style_kwargs=None,
+                      skyview_kwargs={}):
```

Comment:
I don't think this is clean enough. It let the `grid=` through from example code in docs, and then fails the doc build.